virt/backup_restore/virt-installing-configuring-oadp.adoc

```diff
@@ -10,7 +10,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you install the OpenShift API for Data Protection (OADP) by installing the OADP Operator. The Operator installs link:https://{velero-domain}/docs/v1.7/[Velero 1.7].
+You install the OpenShift API for Data Protection (OADP) by installing the OADP Operator. The Operator installs link:https://{velero-domain}/docs/v1.7/[Velero 1.7].
```

Comment:
Only cluster admins can install operators. The change was that non-cluster admins could use OADP. There is a related bug for OADP. That should explain which roles can create the backup and restore CRs.